## ADDIDAS ANALISIS

In [1]:
!wget https://github.com/prayagpadwal/Adidas-Sales-Analysis/raw/main/Adidas%20US%20Sales%20Datasets.xlsx

--2025-09-03 21:44:51--  https://github.com/prayagpadwal/Adidas-Sales-Analysis/raw/main/Adidas%20US%20Sales%20Datasets.xlsx
Resolving github.com (github.com)... 140.82.113.3
Connecting to github.com (github.com)|140.82.113.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://raw.githubusercontent.com/prayagpadwal/Adidas-Sales-Analysis/main/Adidas%20US%20Sales%20Datasets.xlsx [following]
--2025-09-03 21:44:52--  https://raw.githubusercontent.com/prayagpadwal/Adidas-Sales-Analysis/main/Adidas%20US%20Sales%20Datasets.xlsx
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.109.133, 185.199.110.133, 185.199.108.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.109.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 693038 (677K) [application/octet-stream]
Saving to: ‘Adidas US Sales Datasets.xlsx’

Adidas US Sales Dat 100%[===================>] 676.79K  --.-KB/s    in 0.0

In [6]:
import pandas as pd
data = pd.read_excel("Adidas US Sales Datasets.xlsx")

In [8]:
data.head(10)

Retailer  Retailer ID Invoice Date     Region     State      City  \
0  Foot Locker      1185732   2020-01-01  Northeast  New York  New York   
1  Foot Locker      1185732   2020-01-02  Northeast  New York  New York   
2  Foot Locker      1185732   2020-01-03  Northeast  New York  New York   
3  Foot Locker      1185732   2020-01-04  Northeast  New York  New York   
4  Foot Locker      1185732   2020-01-05  Northeast  New York  New York   
5  Foot Locker      1185732   2020-01-06  Northeast  New York  New York   
6  Foot Locker      1185732   2020-01-07  Northeast  New York  New York   
7  Foot Locker      1185732   2020-01-08  Northeast  New York  New York   
8  Foot Locker      1185732   2020-01-21  Northeast  New York  New York   
9  Foot Locker      1185732   2020-01-22  Northeast  New York  New York   

                     Product  Price per Unit  Units Sold  Total Sales  \
0      Men's Street Footwear            50.0        1200     600000.0   
1    Men's Athletic Footwear            50.0        1000     500000.0   
2    Women's Street Footwear            40.0        1000     400000.0   
3  Women's Athletic Footwear            45.0         850     382500.0   
4              Men's Apparel            60.0         900     540000.0   
5            Women's Apparel            50.0        1000     500000.0   
6      Men's Street Footwear            50.0        1250     625000.0   
7    Men's Athletic Footwear            50.0         900     450000.0   
8    Women's Street Footwear            40.0         950     380000.0   
9  Women's Athletic Footwear            45.0         825     371250.0   

   Operating Profit  Operating Margin Sales Method  
0          300000.0              0.50     In-store  
1          150000.0              0.30     In-store  
2          140000.0              0.35     In-store  
3          133875.0              0.35     In-store  
4          162000.0              0.30     In-store  
5          125000.0              0.25     In-store  
6          312500.0              0.50     In-store  
7          135000.0              0.30       Outlet  
8          133000.0              0.35       Outlet  
9          129937.5              0.35       Outlet

0. Cuantas filas y columnas tiene la tabla?

In [7]:
data.shape

(9648, 13)

0.1 Cual es la unidad de observación de la tabla?



>



1. Cuales fueron las ventas totales?

In [13]:
data['Total Sales'].sum()

np.float64(899902125.0)

1.1 Cuanto fue la ganancia operacional total?

In [14]:
data['Operating Profit'].sum()

np.float64(332134761.45000005)

la ganancia operacion fue de 332134761

1.2 Cuanto fue el margen promedio total ?

In [16]:
data['Operating Margin'].mean()

np.float64(0.42299129353233833)

El margen promedio total es de 0.422

1.3 Por que el margen promedio y no el total. Explique con palabras

porque el total del margen promedio nos arrojaría un resultado que no tiene sentido

2. Cuales fueron las ventas totales por Retailer?

In [23]:
TS= data.groupby('Retailer')['Total Sales']. sum()
TS

Retailer
Amazon            77698912.0
Foot Locker      220094720.0
Kohl's           102114753.0
Sports Direct    182470997.0
Walmart           74558410.0
West Gear        242964333.0
Name: Total Sales, dtype: float64

2.1 Cual gue la ganancia operacional total por retailer?

In [19]:
data.groupby('Retailer')['Operating Profit']. sum()

Retailer
Amazon           28818503.31
Foot Locker      80722124.81
Kohl's           36811252.58
Sports Direct    74332954.96
Walmart          25782052.61
West Gear        85667873.18
Name: Operating Profit, dtype: float64

2.3 Tomando los dos resultados de arriba: El ganancia operacional total y las ventas totales divida ambos resultados. Le tiene que dar el margen total de cada Retailer. Haga una Grafica de Barras con el resultado.

In [26]:
from re import M
MT_retailer= data.groupby('Retailer')['Operating Profit']. sum()/data.groupby('Retailer')['Total Sales']. sum()
MT_retailer

Retailer
Amazon           0.370900
Foot Locker      0.366761
Kohl's           0.360489
Sports Direct    0.407369
Walmart          0.345797
West Gear        0.352594
dtype: float64

2.4 Cual fue el margen operacional promedio por retailer?

In [28]:
data. groupby('Retailer')['Operating Margin']. mean()

Retailer
Amazon           0.417871
Foot Locker      0.417933
Kohl's           0.419291
Sports Direct    0.444892
Walmart          0.406534
West Gear        0.417856
Name: Operating Margin, dtype: float64

2.5 Por que los dos resultados anteriores le dieron diferente? Argumente

Porque en el punto 2.3 obtuvimos el margen total y en el 2.4 es el margen promedio

3.1 Retailer que mas unidades totales vende? Saque cuanto vende en unidades cada retailer y busque el mayor

In [33]:
Rt_mayor_unidades_vendidas= data.groupby('Retailer')['Units Sold']. sum().sort_values(ascending=False)
Rt_mayor_unidades_vendidas

Retailer
West Gear        625262
Foot Locker      604369
Sports Direct    557640
Kohl's           287375
Walmart          206225
Amazon           197990
Name: Units Sold, dtype: int64

3.2 Cual fue el producto que mas unidades totales vendió?

In [31]:
data.groupby('Product')['Units Sold']. sum().sort_values(ascending=False)

Product
Men's Street Footwear        593320
Men's Athletic Footwear      435526
Women's Apparel              433827
Women's Street Footwear      392269
Women's Athletic Footwear    317236
Men's Apparel                306683
Name: Units Sold, dtype: int64

3.3 Cual es el metodo de venta que mas ventas totales tuvo y más ventas promedio

In [37]:
Metodo_venttotal= data.groupby('Sales Method')['Total Sales']. sum().sort_values(ascending=False)
Metodo_venttotal

Sales Method
In-store    356643750.0
Outlet      295585493.0
Online      247672882.0
Name: Total Sales, dtype: float64

In [36]:
Met_masventprom= data.groupby('Sales Method')['Total Sales']. mean().sort_values(ascending=False)
Met_masventprom

Sales Method
In-store    204967.672414
Outlet       97908.411063
Online       50659.210882
Name: Total Sales, dtype: float64

3.4 Cual Fue el estado que más ventas totales tuvo ?

In [40]:
Estado_masventas= data.groupby('State')['Total Sales']. sum().sort_values(ascending=False).head(4)
Estado_masventas

State
New York      64229039.0
California    60174133.0
Florida       59283714.0
Texas         46359746.0
Name: Total Sales, dtype: float64

4. Teniendo en cuenta los resultados de arriba: 1. Tome el Retailer que vendio
mas 2. El estado que vendio mas 3.el producto que vendio mas y el metodo de ventas que vendio más 4. Filtre y genere un dataframe con ellos 5. llamelo como un nuevo dataframe

In [41]:
nuevo_dataframe=data[(data["Retailer"]=="West Gear")&(data['State']=="New York")&(data['Product']=="Men's Street Footwear")&(data['Sales Method']=="In-store")]
nuevo_dataframe

Retailer  Retailer ID Invoice Date     Region     State    City  \
1443  West Gear      1185732   2021-03-08  Northeast  New York  Albany   
1449  West Gear      1185732   2021-04-09  Northeast  New York  Albany   
1455  West Gear      1185732   2021-05-08  Northeast  New York  Albany   
1461  West Gear      1185732   2021-06-10  Northeast  New York  Albany   
1467  West Gear      1185732   2021-07-08  Northeast  New York  Albany   
1473  West Gear      1185732   2021-08-09  Northeast  New York  Albany   
1479  West Gear      1185732   2021-09-08  Northeast  New York  Albany   
1485  West Gear      1185732   2021-10-10  Northeast  New York  Albany   
1491  West Gear      1185732   2021-11-09  Northeast  New York  Albany   
1497  West Gear      1185732   2021-12-08  Northeast  New York  Albany   

                    Product  Price per Unit  Units Sold  Total Sales  \
1443  Men's Street Footwear            45.0         870     391500.0   
1449  Men's Street Footwear            45.0         825     371250.0   
1455  Men's Street Footwear            55.0         870     478500.0   
1461  Men's Street Footwear            60.0         875     525000.0   
1467  Men's Street Footwear            65.0         950     617500.0   
1473  Men's Street Footwear            65.0         925     601250.0   
1479  Men's Street Footwear            55.0         850     467500.0   
1485  Men's Street Footwear            60.0         800     480000.0   
1491  Men's Street Footwear            60.0         775     465000.0   
1497  Men's Street Footwear            65.0         875     568750.0   

      Operating Profit  Operating Margin Sales Method  
1443          176175.0              0.45     In-store  
1449          167062.5              0.45     In-store  
1455          215325.0              0.45     In-store  
1461          236250.0              0.45     In-store  
1467          277875.0              0.45     In-store  
1473          270562.5              0.45     In-store  
1479          210375.0              0.45     In-store  
1485          216000.0              0.45     In-store  
1491          209250.0              0.45     In-store  
1497          255937.5              0.45     In-store